cypress/integration/checkLists.ts

```diff
@@ -0,0 +1,31 @@
+describe('On check-lists example', () => {
+  it('checks the bullet when clicked', () => {
+    cy.visit('examples/check-lists')
```

Comment:
Put `visit`s in a `beforeEach`. Every test that will be written `checkList` example will be on that page - even though there's exactly one test for now.